Week-3/Assignment3.1/AddMemoize.js

```diff
@@ -0,0 +1,24 @@
+
+function addNum(a,b) {
+    return a+b;
+}
+
+const memoizeAdd = memoize(addNum);
+function memoize(fn) {
+    const cache = new Map();
+  
+    return function(...args) {
+      const cacheKey = args[0];
+  
+      if (cache.has(cacheKey)) {
+        return cache.get(cacheKey);
+      }
+  
```

Comment:
please fix indentation in the file and do add new empty line at the end of the file.